share/spice/cargo/content.handlebars

```diff
@@ -0,0 +1,3 @@
+<pre>[dependencies]
```

Comment:
Actually, is there only one dependencies for all of these?
I'm wondering if we should move this to the InfoBox like we've done for other IA's e.g. [NPM](https://duckduckgo.com/?q=npm+yolo&ia=software&iax=1)